TL improvement (#523)
07th-mod/higurashi-console-arcs: Update/omot_002_2.txt

//rレナでさえ引きつった笑みのまま、呆れるやら何とやらで複雑な表情だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "レナでさえ引きつった笑みのまま、呆れるやら何とやらで複雑な表情だった。",

> 			NULL, "Even Rena wore a complex expression that also looked astonished with a bitter smile on her face.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even Rena had mixed feelings on her face, looking astonished and wearing a bitter smile.", Line_Normal);
```

	//rそれがなぜか、…ある日を境にふっつりと、距離を置くようになったのだ。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "それがなぜか、…ある日を境にふっつりと、距離を置くようになったのだ。",

> 				NULL, "I don't know why but... one day, she put up a wall and distanced herself from us.", Line_Normal);

Reviewer: ```suggestion
				NULL, "I don't know why, but... one day, she put up a wall and distanced herself from us.", Line_Normal);
```
